packages/driver/src/cypress/error_messages.ts

```diff
@@ -989,29 +992,29 @@ export default {
         If so, increase \`redirectionLimit\` value in configuration.`
     },
     cross_origin_load_timed_out ({ ms, configFile, crossOriginUrl, originPolicies }) {
-      return stripIndent`\
-
-        Timed out after waiting \`${ms}ms\` for your remote page to load on origin(s):
-
-        - ${originPolicies.map((originPolicy) => `\`${originPolicy}\``).join('\n       -')}
+      return {
+        message: stripIndent`\
+          Timed out after waiting \`${ms}ms\` for your remote page to load on origin(s):
 
-        Your page did not fire its \`load\` event within \`${ms}ms\`.
+          - ${originPolicies.map((originPolicy) => `\`${originPolicy}\``).join('\n       -')}
 
-        A cross origin request for \`${crossOriginUrl.href}\` was detected.
+          A cross origin request for \`${crossOriginUrl.href}\` was detected.
 
-        A command that triggers cross origin navigation must be immediately followed by a ${cmd('origin')} command:
+          A command that triggers cross origin navigation must be immediately followed by a ${cmd('origin')} command:
 
-        \`\`\`
-        cy.origin('${crossOriginUrl.originPolicy}', () => {
-          <commands targeting ${crossOriginUrl.origin} go here>
-        })
-        \`\`\`
+          \`\`\`
+          cy.origin('${crossOriginUrl.originPolicy}', () => {
+            <commands targeting ${crossOriginUrl.origin} go here>
+          })
+          \`\`\`
 
-        If the cross origin request was an intermediary state, you can try increasing the \`pageLoadTimeout\` value in ${formatConfigFile(configFile)} to wait longer.
+          If the cross origin request was an intermediary state, you can try increasing the \`pageLoadTimeout\` value in ${formatConfigFile(configFile)} to wait longer.
 
-        Browsers will not fire the \`load\` event until all stylesheets and scripts are done downloading.
+          Browsers will not fire the \`load\` event until all stylesheets and scripts are done downloading.
```

Comment:
was this meant to be another reason an increased timeout might be needed?

Reply:
I think so? mostly this message was cribbed from here, with added cy.origin stuff added:
https://github.com/cypress-io/cypress/blob/feature-multidomain/packages/driver/src/cypress/error_messages.ts#L973-L984

Reply:
I wouldn't be opposed to removing this line and the next.